packages/utils/src/color/Color.ts

```diff
@@ -0,0 +1,305 @@
+    {
+        if (this._value !== value)
+        {
+            this._value = value;
+            this.normalize();
+        }
+    }
+    get value(): ColorSource
+    {
+        return this._value;
+    }
+
+    /**
+     * Get RGBA color object.
+     * @example
+     * import { utils } from 'pixi.js';
+     * new utils.Color('white').toRgb(); // returns { r: 1, g: 1, b: 1, a: 1 }
+     */
+    toRgba(): RgbaColor
+    {
+        const [r, g, b, a] = this._components;
+
+        return { r, g, b, a };
+    }
+
+    /**
+     * Get RGB color object.
+     * @example
+     * import { utils } from 'pixi.js';
+     * new utils.Color('white').toRgb(); // returns { r: 1, g: 1, b: 1 }
+     */
+    toRgb(): RgbColor
+    {
+        const [r, g, b] = this._components;
+
+        return { r, g, b };
+    }
+
+    /** Get the color as a CSS-style rgba string: `rgba(255,255,255,1.0)`. */
+    toRgbaString(): string
+    {
+        const [r, g, b, a] = this._components;
+
+        return `rgba(${Math.round(r * 255)},${Math.round(g * 255)},${Math.round(b * 255)},${a})`;
+    }
+
+    /**
+     * Converts color to an [R, G, B] array of normalized floats (numbers from 0.0 to 1.0).
+     * @example
+     * import { utils } from 'pixi.js';
+     * new utils.Color('white').toRgbArray(); // returns [1, 1, 1]
+     * @param {number[]|Float32Array} [out] - Output array
+     */
+    toRgbArray<T extends (number[] | Float32Array) = number[]>(out?: T): T
+    {
+        out = out ?? [] as T;
+        const [r, g, b] = this._components;
+
+        out[0] = r;
+        out[1] = g;
+        out[2] = b;
+
+        return out;
+    }
+
+    /**
+     * Converts a color to a hexadecimal number.
+     * @example
+     * import { utils } from 'pixi.js';
+     * new utils.Color('white').toNumber(); // returns 16777215
+     */
+    toNumber(): number
+    {
+        return this._int;
+    }
+
+    /**
+     * Converts a hexadecimal color number to a string.
+     * @example
+     * import { utils } from 'pixi.js';
+     * new utils.Color('white').toString(); // returns "#ffffff"
+     */
+    toString(): string
+    {
+        const hexString = this._int.toString(16);
+
+        return `#${'000000'.substring(0, 6 - hexString.length) + hexString}`;
+    }
+
+    /**
+     * Convert to a hexidecimal string with alpha.
+     * @example
+     * import { utils } from 'pixi.js';
+     * new utils.Color('white').toHexString(); // returns "#ffffffff"
+     */
+    toHexString(): string
+    {
+        const alphaValue = Math.round(this._components[3] * 255);
+        const alphaString = alphaValue.toString(16);
+
+        return this.toString() + '00'.substring(0, 2 - alphaString.length) + alphaString;
+    }
+
+    /**
+     * Set alpha, suitable for chaining.
+     * @param alpha
+     */
+    setAlpha(alpha: number): this
+    {
+        this._components[3] = alpha;
+
+        return this;
+    }
+
+    /** Get the alpha amount */
+    toAlpha(): number
```

Comment:
Maybe `getAlpha`?

Reply:
I was thinking about adding getters for alpha, r, g, b intead